nilearn/decoding/decoder.py

```diff
@@ -223,6 +243,14 @@ class _BaseDecoder(LinearModel, RegressorMixin, CacheMixin):
         or have no effect. See scikit-learn documentation for more information,
         for example: https://scikit-learn.org/stable/modules/grid_search.html
 
+    do_clustering: boolean. Default False
+        Whether to do a ReNA clustering as first step of fit to agglomerate similar
+        features together and reduce their overall number.
+
+    clustering_percentile: int, float, optional, in the [0, 100] Default: 10.
+        Percentile of features to keep if clustering is performed.
+        ReNA is typically efficient at reducing the number of features by 10.
```

Comment:
I guess here you mean 10 folds @thomasbazeille ?

Reply:
no I mean clustering in order to reduce the number of features 
from `n` voxels
to `n * clustering_percentile/100` clusters 

I'm not sure how to make it clearer though if you find it ill-explained. Would you have a suggestion ?